BCH-Wunder-Project-team-4/Wunder.io
# Content

## Link to ticket:



## Link to feature environment:
http://localhost:3000/

## Changes proposed in this PR:

Some minor changes in the config 
New database with content 

## How to test:

**BEFORE STARTING CREATE BACKUP** `lando db-export`  **(Just in case)**

`git fetch origin content`
`git checkout content`
`lando drush cim`
`lando db-import DBnewContent.sql.gz`

check the localhost.... 
I'm not sure if pictures also will be in this DB 
If u switch to finish language there should be almost all the same content except events. Events don't support translation at all
